100mslive/100ms-flutter
# 1230/advanced camera controls android

# Description

Added camera controls method to capture image at high resolution 

Closes:
- #1230 

## Additional context

How to test:

- Click on three dots button on local peer tile.
- Click on  `local Image Capture`.
- Should get a popup with an option to share the image.
- Check the image in device for resolution, go to `android/data/live.hms.flutter/files/images`

### Pre-launch Checklist

- [ ] The [Documentation] is updated accordingly, or this PR doesn't require it.


[Documentation]: https://www.100ms.live/docs
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
